nautobot/docs/configuration/authentication/ldap.md

```diff
@@ -126,6 +126,31 @@ AUTH_LDAP_USER_ATTR_MAP = {
 }
 ```
 
+The following snippet shows how to search for users in multiple LDAP groups:
+
+- Define the user-groups in *.env file (delimiter `';'`)
+
+    ```bash
+    # Groups to search for user objects. "(sAMAccountName=%(user)s),..."
+    NAUTOBOT_AUTH_LDAP_USER_SEARCH_DN=OU=IT-Admins,OU=special-users,OU=Acme-User,DC=Acme,DC=local;OU=Infrastruktur,OU=IT,OU=my-location,OU=User,OU=Acme-User,DC=Acme,DC=local
+    ```
+
+- Import LDAPSearchUnion in `nautobot_config.py`
+
+    ```python
+    from django_auth_ldap.config import ..., LDAPSearchUnion
+    ```
+
+- In `nautobot_config.py` replace the AUTH_LDAP_USER_SEARCH command from above with:
+
+    ```bash
+    user_search_dn_list = str(AUTH_LDAP_USER_SEARCH_DN).split(";")
+    ldapsearch_objects = []
+    for sdn in user_search_dn_list:
+        ldapsearch_objects.append(LDAPSearch(sdn.strip(), ldap.SCOPE_SUBTREE, "(sAMAccountName=%(user)s)"))
+    AUTH_LDAP_USER_SEARCH = LDAPSearchUnion(*ldapsearch_objects)
+    ```
```

Comment:
This doesn't render properly [on ReadTheDocs](https://nautobot--1080.org.readthedocs.build/en/1080/configuration/authentication/ldap/#user-authentication) (you can check the rendering locally with `mkdocs serve` then pointing your browser to `http://localhost:8001`)

Possibly the following might render correctly, but please try it out:

````suggestion
Define the user-groups in your environment, such as a *.env file (delimiter `';'`):

```no-highlight
# Groups to search for user objects. "(sAMAccountName=%(user)s),..."
NAUTOBOT_AUTH_LDAP_USER_SEARCH_DN=OU=IT-Admins,OU=special-users,OU=Acme-User,DC=Acme,DC=local;OU=Infrastruktur,OU=IT,OU=my-location,OU=User,OU=Acme-User,DC=Acme,DC=local
```

Import LDAPSearchUnion in `nautobot_config.py`, and replace the AUTH_LDAP_USER_SEARCH command from above:

```python
from django_auth_ldap.config import ..., LDAPSearchUnion

# ...

user_search_dn_list = str(AUTH_LDAP_USER_SEARCH_DN).split(";")
ldapsearch_objects = []
for sdn in user_search_dn_list:
    ldapsearch_objects.append(LDAPSearch(sdn.strip(), ldap.SCOPE_SUBTREE, "(sAMAccountName=%(user)s)"))
AUTH_LDAP_USER_SEARCH = LDAPSearchUnion(*ldapsearch_objects)
```
````